src/userApps.ts

```diff
@@ -146,4 +162,25 @@ export class UserApps {
         if (!res.verified) throwError('JWT failed verification', SdkErrors.JwtNotValid);
         return res;
     }
+
+    static async verifyPrivateKey(accountName: string, keyManager?: KeyManager): Promise<boolean> {
```

Comment:
1. Let's rename this to makke it clearer `verifyKeyExistsForApp(appAccountName: string, keyManager: KeyManager)
2. Also make the KeyManagerLevel one of the arguments to the function so that we can use this for all different KeyManagerLevels.